apps/store/src/components/ComparisonTable/ComparisonTable.css.ts

```diff
@@ -83,13 +83,20 @@ export const collapsibleContent = style({
   maxWidth: '45ch',
 })
 
-export const triggerIcon = style({
-  flexShrink: 0,
-  width: '1rem',
-  aspectRatio: '1',
-  transition: 'transform 300ms',
-  selectors: {
-    '[data-state=open] &': { transform: 'rotate(180deg)' },
-    '[data-state=closed] &': { transform: 'rotate(0deg)' },
+export const triggerIconVariants = styleVariants({
+  base: {
+    flexShrink: 0,
+    width: '1rem',
+    aspectRatio: '1',
+  },
+  closed: {
+    selectors: {
+      [`${collapsibleTrigger}[data-state=open] &`]: { display: 'none' },
+    },
+  },
+  open: {
+    selectors: {
+      [`${collapsibleTrigger}[data-state=closed] &`]: { display: 'none' },
+    },
```

Comment:
Nit

```suggestion
const triggerIconBase = style({
    flexShrink: 0,
    width: '1rem',
    aspectRatio: '1',
}}
export const triggerIconVariants = styleVariants({
  closed: [triggerIconBase, {
    selectors: {
      [`${collapsibleTrigger}[data-state=open] &`]: { display: 'none' },
    },
  }],
  open: [triggerIconBase, {
    selectors: {
      [`${collapsibleTrigger}[data-state=closed] &`]: { display: 'none' },
    }],
```

Then you could:

```
 <PlusIcon
    color={tokens.colors.textSecondary}
    className={triggerIconVariants.closed}
  />
```

Any particular reason why it was chosen to add both classes with `clsx` instead of `vanilla extract`? 🤔